src/I18n/Number.php

```diff
@@ -276,6 +276,10 @@ public static function formatter($options = [])
             $type = NumberFormatter::CURRENCY;
         }
 
+        if (!empty($options['type']) && $options['type'] === 'ordinal') {
```

Comment:
Right. But, currently the only option for `type` is to pass in a `'currency`' string, otherwise it defaults to `DECIMAL`. There is no `if (is_int()) { //set format }`. We *could* check to see if `type` is an int and use that as the type.